openlibrary/macros/SubjectTags.html

```diff
@@ -0,0 +1,22 @@
+$def with(work, tags=[])
+
+$def render_subjects(label, subjects, prefix=""):
+  $if subjects:
+    <div class="section link-box">
+      <h6>$label</h6>
+      <span>
+        $for subject in subjects:
+          <a href="/subjects/$prefix$utf8(subject.lower().replace(' ', '_').replace(',',''))">$subject</a>$cond(not loop.last,",","")
+      </span>
+    </div>
+
+$if work:
+  $for tag in tags:
+    $if tag=="Subjects":
+      $:render_subjects("Subjects", work.get_subjects())
```

Comment:
i18n "Subjects" (and on the other lines)

Reply:
done within render_subjects as $_(label)

Reply:
Needs to be compile time constant